Makefile

```diff
@@ -229,3 +242,24 @@ src/allmydata/_version.py:
 
 .tox: tox.ini setup.py
 	tox --notest -p all
+
+
+# BBB: Python 3 porting targets
```

Comment:
What does BBB mean?

Reply:
It's a convention I picked up long ago for backwards compatible code.  It often means, "remove this when we drop compatibility".  I often end up using it to notate stuff that should be removed when some particular larger work is completed.

I've seen it in other folk's code for so long I just assumed it was wide spread, but I just tried to google it and came up short.  Had no idea this wasn't universal.  ;-)